### PR TITLE
Add get volume function

### PR DESCRIPTION
**Pull request by Liborsaf**

### CODE REVIEW

**Comment by alexmohr** [comment]
Thanks for the contribution. I removed travis and added github ci instead. In turn I had to fix some new linter warnings, so you have to fix some conflicts now. Sorry :/
Also please add a unit test for the new method

**Comment by Liborsaf** [comment]
> Thanks for the contribution. I removed travis and added github ci instead. In turn I had to fix some new linter warnings, so you have to fix some conflicts now. Sorry :/ Also please add a unit test for the new method

No problem, will look into that in few minutes :)

**Comment by Liborsaf** [comment]
Not very experienced in doing unit tests, so hopefully this is good :D

File: sonyapilib/device.py [hunk 1]
```diff
@@ -359,12 +360,25 @@ def _parse_dmr(self, data):
...
                 service_id = service.find(
                     f"{URN_UPNP_DEVICE}serviceId")
-                if "urn:upnp-org:serviceId:AVTransport" not in service_id.text:
-                    continue
-                transport_location = service.find(
-                    f"{URN_UPNP_DEVICE}controlURL").text
-                self.av_transport_url = f"{lirc_url.scheme}://{lirc_url.netloc.split(':')[0]}"\
-                    f":{self.dmr_port}{transport_location}"
+                service_id_text = service_id.text
+                # print(service_id_text)
```
Comment (alexmohr) [inline sonyapilib/device.py]: remove comment (all commented out print statements you added )

**Review by alexmohr — CHANGES_REQUESTED** [review]
Thanks, the test is good just one more minor finding and then this is good to go :)

File: sonyapilib/device.py (hunk 1 above)
Comment (Liborsaf) [reply sonyapilib/device.py]: Done 👍🏻

**Pull request merged**